src/components/common/FooterBtn/FooterBtn.Style.ts

```diff
@@ -12,17 +12,17 @@ export const footerBtnStyle = (
 		display: flex;
 		align-items: center;
 		justify-content: center;
-		width: ${isPositionStatic || isSquare ? "100%" : "22.5rem"};
+		width: 100%;
 		padding: ${!isSquare && !isPositionStatic && "1rem 1.5rem"};
 		gap: ${!isSquare && "0.5rem"};
 		position: ${!isPositionStatic && "fixed"};
 		bottom: ${!isPositionStatic && 0};
-		background-color: ${!transparent && "white"};
+		background-color: ${!transparent && "#fff"};
 		left: ${!isPositionStatic && "50%"};
 		transform: ${!isPositionStatic && "translateX(-50%)"};
 
 		& > button {
-			width: 100%;
+			width: 19.5rem;
```

Comment:
버튼 가로 사이즈를 19.5rem으로 했을때

사진처럼 바텀시트 하단에 버튼이 있는 경우 버튼이 끝까지 채워지지 않습니다

<img width="640" alt="스크린샷 2024-07-24 오후 3 20 08" src="https://github.com/user-attachments/assets/0b86b263-2909-4e2f-80d7-2a60bd24549d">
<img width="399" alt="스크린샷 2024-07-24 오후 3 28 30" src="https://github.com/user-attachments/assets/6c698645-2799-4b43-a425-c88c194364fd">





바텀시트가 항상 가로로 가득 채워지는 형태니 

버튼도 isSquare일때는 100%로 하는게 좋을 것 같아요

```suggestion
			width: ${isSquare ? "100%" : "19.5rem"};
```

이 경우 제외하고 다른 페이지나 모달에서는 문제 없었습니다 !